src/lib/browser/Adblocker.ts

```diff
@@ -1,8 +1,9 @@
 import { request } from 'undici';
 
+import { report } from '../../helpers/errorReporting';
 import { log as mainLog } from '../../helpers/logger';
 
-// TO DO: copy this at compile time.
+// TO DO: Cronjob to update this list on our servers
```

Comment:
I would keep the old todo: it's easier, consumes less bandwidth and it's not important if the list gets slightly outdated.
Otherwise good catch!